tests/php/Security/Member_GroupSetTest.php

```diff
@@ -34,10 +33,12 @@ public function testForForeignIDPlaceholders(bool $config, bool $useInt, bool $e
         $expectedIDs = $useInt
             ? array_unique(array_merge($groups1->column('ID'), $groups2->column('ID')))
             : [];
-        $this->assertEqualsCanonicalizing($expectedIDs, $newGroupList->column('ID'));
```

Comment:
No idea why though this unit test was failing, almost like assertEqualsCanonicalizing() just wasn't doing what it was supposed to. I just sorted things the manual way and the test passes.